data/json/items/comestibles/brewing.json

```diff
@@ -141,6 +141,73 @@
     "flags": [ "NUTRIENT_OVERRIDE" ],
     "comestible_type": "DRINK"
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "brew_sake",
+    "looks_like": "brew_vodka",
+    "name": { "str_sp": "sake mash" },
+    "description": "A fine mix of water, koji rice and yeast, ready to be fermented into sake once set in a fermenting vat.",
+    "weight": "33 g",
+    "color": "white",
+    "container": "bottle_glass",
+    "flags": [ "TRADER_AVOID", "NUTRIENT_OVERRIDE" ],
+    "symbol": "~",
+    "quench": 4,
+    "calories": 69,
+    "fun": -5,
+    "price_postapoc": 10,
+    "material": [ "veggy", "water" ],
+    "volume": "250 ml",
+    "charges": 7,
+    "phase": "liquid",
+    "comestible_type": "DRINK",
+    "brewable": { "time": "7 days", "results": [ "sake_unfiltered", "koji", "yeast" ] }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "brew_soy_sauce",
+    "name": { "str_sp": "unfermented soy sauce" },
+    "description": "Brine with mashed soybeans, wheat flour and koji mold added to it.  If you want to make soy sauce from this, next step would be to let it ferment.",
+    "weight": "24 g",
+    "color": "white",
+    "container": "bottle_glass",
+    "flags": [ "TRADER_AVOID", "NUTRIENT_OVERRIDE" ],
+    "symbol": "~",
+    "quench": -20,
+    "healthy": -1,
+    "fun": -15,
+    "calories": 17,
+    "price_postapoc": 0,
+    "material": [ "veggy", "water" ],
+    "volume": "250 ml",
+    "charges": 10,
+    "phase": "liquid",
+    "comestible_type": "DRINK",
+    "freezing_point": -30,
+    "brewable": { "time": "21 days", "results": [ "soy_sauce_unstrained" ] }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "soy_sauce_unstrained",
+    "name": { "str_sp": "unstrained soy sauce" },
+    "description": "Nearly finished soy sauce that still has some solid particles floating in it.  Still needs to be strained, and also pasteurized to get rid of harmful bacteria.",
+    "weight": "24 g",
+    "color": "white",
+    "container": "bottle_glass",
+    "symbol": "~",
+    "quench": -20,
+    "healthy": -1,
```

Comment:
If there are harmful bacteria in it, I would make the health penalty a bit bigger and maybe even give it a chance to induce vomiting.